packaging/arch/regenerate_images

```diff
@@ -3,15 +3,25 @@
 # find out all installed kernels
 mapfile -d '' kernels < <(find /usr/lib/modules -maxdepth 1 -type d ! -name "modules" -print0)
 
+# paths to boot partition for the initramfs image and kernel
+boot_paths=("/efi" "/boot")
```

Comment:
`/boot/efi` is technically also a valid boot partition path for systemd and from the mkinitcpio presets it appears Arch Linux assumes `/efi` by default. Since paths should be distribution dependent, choosing not to include `/boot/efi`